k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml

```diff
@@ -89,20 +89,26 @@
     "sha256:131ece0637b29231470cfaa04690c2966a2e0b147d3c9df080a0857b78982410": ["v20210810-g820a21a74"]
 
 # fastcgi HTTP server image for e2e tests
-# https://github.com/kubernetes/ingress-nginx/tree/master/images/fastcgi-helloserver
+# https://github.com/kubernetes/ingress-nginx/tree/main/images/fastcgi-helloserver
 - name: e2e-test-fastcgi-helloserver
   dmap:
     "sha256:723b8187e1768d199b93fd939c37c1ce9427dcbca72ec6415f4d890bca637fcc": ["v20200627-g1fcf4444c"]
 
 # httpbin image for e2e tests
-# https://github.com/kubernetes/ingress-nginx/tree/master/images/httpbin
+# https://github.com/kubernetes/ingress-nginx/tree/main/images/httpbin
 - name: e2e-test-httpbin
   dmap:
     "sha256:c6372ef57a775b95f18e19d4c735a9819f2e7bb4641e5e3f27287d831dfeb7e8": ["v20200627-gfc91afac8"]
     "sha256:12f18b0ce1fa12e40a29b4b142cbfc6b909b65a0be02421bc3d1889c824747f7": ["v20200808-gc500bd4b3"]
 
 # kube-webhook-certgen
-# https://github.com/kubernetes/ingress-nginx/tree/master/images/kube-webhook-certgen
+# https://github.com/kubernetes/ingress-nginx/tree/main/images/kube-webhook-certgen
 - name: kube-webhook-certgen
   dmap:
     "sha256:f3b6b39a6062328c095337b4cadcefd1612348fdd5190b1dcbcb9b9e90bd8068": ["v1.0"]
+
+# custom-error-pages
+# https://github.com/kubernetes/ingress-nginx/tree/main/images/custom-error-pages
+- name: kube-webhook-certgen
```

Comment:
Hum, here probably is "nginx-errors" (you should use the same name on https://github.com/kubernetes/ingress-nginx/pull/7625/files#diff-2fd4fb6ee7bf7cd038c9d88c82b182ae7b1de5f4457b7986f872add33c40f0f0R39)